.gitignore

```diff
@@ -44,3 +44,6 @@ configs
 # Private testnet artifacts
 *-keep-client-deployment-bundle.tar.gz
 keep-client-snapshot.tar
+solidity/random-beacon/cache
+solidity/random-beacon/yarn-error.log
+solidity/random-beacon/export.json
```

Comment:
What if we define it in a more generic way, to cover also what we will have in `solidity/keep-ecdsa/...`?

```suggestion

# Yarn
yarn-error.log

# Hardhat
cache/
export.json
```

Reply:
Sure, done 75bb0f7